_posts/2023-07-17-video-library-launching.md

```diff
@@ -0,0 +1,83 @@
+---
+layout: post
+title: "Launching the Open Seeds Video Library"
+authors:
+- bebatut
+image: https://images.pexels.com/photos/114820/pexels-photo-114820.jpeg
+photos:
+  name: Mike Bird
+  url: https://www.pexels.com/photo/elvis-presley-digital-wallpaper-114820/
+  licence: CC-BY
+---
+
+{% assign cohort-calls = 0 %}
+{% for w in site.data.ols-1-schedule.weeks %}
+    {% for c in w[1].calls %}
+        {% if c.type != "Mentor-Mentee" %}
+            {% assign cohort-calls = cohort-calls | plus: 1 %}
+        {% endif %}
+    {% endfor %}
+{% endfor %}
+{% for w in site.data.ols-2-schedule.weeks %}
+    {% for c in w[1].calls %}
+        {% if c.type != "Mentor-Mentee" %}
+            {% assign cohort-calls = cohort-calls | plus: 1 %}
+        {% endif %}
+    {% endfor %}
+{% endfor %}
+{% for w in site.data.ols-3-schedule.weeks %}
+    {% for c in w[1].calls %}
+        {% if c.type != "Mentor-Mentee" %}
+            {% assign cohort-calls = cohort-calls | plus: 1 %}
+        {% endif %}
+    {% endfor %}
+{% endfor %}
+{% for w in site.data.ols-4-schedule.weeks %}
+    {% for c in w[1].calls %}
+        {% if c.type != "Mentor-Mentee" %}
+            {% assign cohort-calls = cohort-calls | plus: 1 %}
+        {% endif %}
+    {% endfor %}
+{% endfor %}
+{% for w in site.data.ols-5-schedule.weeks %}
+    {% for c in w[1].calls %}
+        {% if c.type != "Mentor-Mentee" %}
+            {% assign cohort-calls = cohort-calls | plus: 1 %}
+        {% endif %}
+    {% endfor %}
+{% endfor %}
+{% for w in site.data.ols-6-schedule.weeks %}
+    {% for c in w[1].calls %}
+        {% if c.type != "Mentor-Mentee" %}
+            {% assign cohort-calls = cohort-calls | plus: 1 %}
+        {% endif %}
+    {% endfor %}
+{% endfor %}
+{% for w in site.data.ols-7-schedule.weeks %}
+    {% for c in w[1].calls %}
+        {% if c.type != "Mentor-Mentee" %}
+            {% assign cohort-calls = cohort-calls | plus: 1 %}
+        {% endif %}
+    {% endfor %}
+{% endfor %}
+
+{% assign library = site.data.library %}
+{% assign video_nb = 0 %}
+{% assign all-speakers = '' %}
+{% for topic in library %}
+    {% for tag in topic[1] %}
+        {% assign video_nb = video_nb | plus: tag[1].talks.size %}
+        {% for v in tag[1].talks %}
+            {% capture all-speakers %}{{ all-speakers }},{{ v.speakers | join:"," }}{% endcapture %}
+        {% endfor %}
+    {% endfor %}
+{% endfor %}
+{% assign speaker_nb = all-speakers | remove_first: ',' | split: "," | uniq | sort | size %}
+
+Since its launch in 2020, OLS ran **7 cohorts**, **{{ cohort-calls }} cohort calls** with **{{ video_nb }} talks** by **{{ speaker_nb }} speakers**. The cohort calls are all available on [OLS YouTube channel]({{ site.youtube }}) and linked in the dedicated cohort schedule. But **all these talks were alo interesting individually and are currently hard to find**.
```

Comment:
```suggestion
Since its launch in 2020, OLS ran **7 cohorts**, **{{ cohort-calls }} cohort calls** with **{{ video_nb }} talks** by **{{ speaker_nb }} speakers**. The cohort calls are all available on [OLS YouTube channel]({{ site.youtube }}) and linked in the dedicated cohort schedule. But **all these talks were also interesting individually and are currently hard to find**.
```